mlflow/resource_webhook_test.go

```diff
@@ -0,0 +1,229 @@
+package mlflow
+
+import (
+	"testing"
+
+	"github.com/databrickslabs/terraform-provider-databricks/qa"
+	"github.com/stretchr/testify/assert"
+)
+
+var (
+	test_wh_job = Webhook{
+		Events:      []string{"TRANSITION_REQUEST_CREATED"},
+		Description: "Job webhook trigger",
+		Status:      "ACTIVE",
+		JobSpec: &JobSpec{
+			JobID:        "1234",
+			WorkspaceURL: "https://test.cloud.databricks.com",
+			AccessToken:  "dapi1234",
+		},
+	}
+	test_wh_id          = "12345"
+	test_wh_job_with_id = Webhook{
+		ID:          test_wh_id,
+		Events:      []string{"TRANSITION_REQUEST_CREATED"},
+		Description: "Job webhook trigger",
+		Status:      "ACTIVE",
+		JobSpec: &JobSpec{
+			JobID:        "1234",
+			WorkspaceURL: "https://test.cloud.databricks.com",
+			AccessToken:  "dapi1234",
+		},
+	}
+	test_wh_hcl = `
+	events = ["TRANSITION_REQUEST_CREATED"]
+	description = "Job webhook trigger"
+	status = "ACTIVE"
+	job_spec {
+		job_id = "1234"
+		workspace_url = "https://test.cloud.databricks.com"
```

Comment:
Oh, we also need to specify workspace url. Very odd. Perhaps later versions should add workspace url of configured client and make a token

Reply:
job could be in another workspace, but maybe we can mark it as omitempty, and fill URL/token if they aren't specified